### PR TITLE
[WIP] - Translate layout status

### PR DESCRIPTION
**Pull request by ndufreche**
- [x] Add en xlf file
- [x] Update default local and default lang
- [x] Translate layout status

### CODE REVIEW

**Comment by ndufreche** [comment]
This PR is to fix backbee/BbCoreJS#355

**Comment by eric-chau** [comment]
:+1: good job

File: Site/Layout.php [hunk 1]
```diff
@@ -747,11 +747,12 @@ public function getWokflowStates()
                 );
             }
         }
+        $translator = $this->getApplication()->getContainer()->get('translator');
```
Comment (mickaelandrieu) [inline Site/Layout.php]: I don't think you have access to the application in Site\Layout() object.

File: Site/Layout.php [hunk 2]
```diff
@@ -749,9 +749,9 @@ public function getWokflowStates()
         }
 
         $workflowStates = array_merge(
-            array('0' => array('label' => 'Hors ligne', 'code' => '0')),
+            array('0' => array('label' => 'Offline', 'code' => '0')),
```
Comment (mickaelandrieu) [inline Site/Layout.php]: this will broke the french translations.

**Comment by mickaelandrieu** [comment]
Thank you @ndufreche

**Comment by ndufreche** [comment]
thanks

**Pull request merged**